app/models/server/models/Users.js

```diff
@@ -136,7 +136,7 @@ export class Users extends Base {
 	}
 
 	findOneOnlineAgentByUsername(username, options) { // TODO:: Create class Agent
-		const query = queryStatusAgentOnline({ username });
+		const query = queryStatusAgentOnline({ username: { $in: username } });
```

Comment:
In this case, the `username` param is an array, isn't it?
If that's the case, can you rename it to `usernames` to avoid confusion? :)

Reply:
```suggestion
		const query = queryStatusAgentOnline({ username: { $in: usernames } });
```

Reply:
Done